scripts/article.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+print_help() {
+	echo "Use a secure tunnel to access or modify articles stored in the postgres databse from the command line"
+	echo
+	echo "Usage: article [OPTION]... [URL]"
+	echo
+	echo "OPTIONS:"
+	echo "  ACTION OPTIONS"
+	echo "  -h         display this help & exit"
+	echo "  -p         print data & exit"
+	echo "  -u         upload data to url & exit"
+	echo "             requires TEXT, TITLE, URL"
+	echo "  DATA OPTIONS"
+	echo "  -D <string> DESCRIPTION string (markdown)"
+	echo "  -d <file>   DESCRIPTION target file (markdown)"
+	echo "  -S <string> SUMMARY string (markdown)"
+	echo "  -s <file>   SUMMARY target file (markdown)"
+	echo "  -I <string> TITLE string"
+	echo "  -i <file>   TITLE target file"
+	echo "  -T <string> TEXT string (markdown)"
+	echo "  -t <file>   TEXT target file (markdown)"
+	echo
+}
+
+print_vars() {
+	echo "TEXT: $text"
+	echo "SUMMARY: $summary"
+	echo "DESCRIPTION: $description"
+	echo "TITLE: $title"
+	echo "URL: $url"
+}
+
+create_proxy() {
+	# prepare proxy program cleanup
+	cleanup() {
+		# kill proxy
+		kill -INT "$proxy_program_PID"
+		sleep 0.4
+	}
+	trap cleanup EXIT
+	# create proxy to connect to database on port 5432
```

Comment:
Further below in the deploy-database file, you put spaces between your comments. I do suggest adding extra space where it makes sense as long as your compiler removes blank spaces.